app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Models/Place.java

```diff
@@ -1,58 +1,62 @@
 package ithaca_transit.android.cornellappdev.com.ithaca_transit.Models;
 
-public final class Place extends LocationObject{
+import com.google.gson.annotations.SerializedName;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.Optional;
+
+import ithaca_transit.android.cornellappdev.com.ithaca_transit.R;
+import ithaca_transit.android.cornellappdev.com.ithaca_transit.Singleton.Repository;
+
+public final class Place{
     private String detail;
-    private String detailKey;
+
+    @SerializedName("lat")
+    private Double latitude;
+
+    @SerializedName("long")
+    private Double longitude;
     private String name;
-    private String nameKey;
     private String placeID;
-    private String placeIDKey;
+    private String type;
```

Comment:
shouldn't all of these be prefaced by `m`, `mType, mLongitude`, etc

Reply:
That's a product of the FutureNova library, it matches JSON variable names right to the exact variable names in the class, so we're not prefacing these.